azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -279,599 +278,701 @@
     //
 
     /**
-     * The first stage of a virtual machine definition.
-     */
-    interface DefinitionBlank extends GroupableResource.DefinitionWithRegion<DefinitionWithGroup> {
-    }
-
-    /**
-     * The stage of the virtual machine definition allowing to specify the resource group.
-     */
-    interface DefinitionWithGroup extends GroupableResource.DefinitionWithGroup<DefinitionWithNetwork> {
-    }
-
-    /**
-     * The stage of the virtual machine definition allowing to specify virtual network for the new primary network
-     * interface or to use a creatable or existing network interface.
-     */
-    interface DefinitionWithNetwork extends DefinitionWithPrimaryNetworkInterface {
-        /**
-         * Create a new virtual network to associate with the virtual machine's primary network interface, based on
-         * the provided definition.
-         *
-         * @param creatable a creatable definition for a new virtual network
-         * @return the next stage of the virtual machine definition
-         */
-        DefinitionWithPrivateIp withNewPrimaryNetwork(Network.DefinitionStages.WithCreate creatable);
-
-        /**
-         * Creates a new virtual network to associate with the virtual machine's primary network interface.
-         * <p>
-         * the virtual network will be created in the same resource group and region as of virtual machine, it will be
```

Comment:
Capitalization -- this is a full sentence in the description so should start with a Capital.